src/Engine/Renderer/Material/PlainMaterial.hpp

```diff
@@ -0,0 +1,123 @@
+#ifndef RADIUMENGINE_PLAINMATERIAL_HPP
+#define RADIUMENGINE_PLAINMATERIAL_HPP
+
+#include <Engine/RaEngine.hpp>
+
+#include <map>
+#include <string>
+
+#include <Core/Utils/Color.hpp>
+#include <Engine/Renderer/Material/Material.hpp>
+#include <Engine/Renderer/Texture/Texture.hpp>
+
+namespace Ra {
+namespace Engine {
+
+class ShaderProgram;
+
+/**
+ * Implementation of the Plain Material BSDF.
+ * This material implements a lambertian diffuse BSDF.
+ * This material could not be loaded from a file and must be defined and associated to
+ * renderobjects programatically.
+ *
+ * This Material is parameterized either globally, by setting the m_color base color or locally,
+ * by setting a vertex color attribute that define the per vertex color and setting m_perVertexColor
+ * to true.
+ *
+ * If the lighting must be disable for this material, resulting in constant color for all the
+ * associated RenderObject, set the m_shaded to false. Set it to true to obtain a lambertian diffuse
+ * BSDF.
```

Comment:
As discussed, `Plain` should always be Plain, and the Lambertian case moved to a dedicated material.